runtime/interpreter/value.go

```diff
@@ -18594,6 +18612,138 @@ func (v LinkValue) ChildStorables() []atree.Storable {
 	}
 }
 
+// PublishedValue
+
+type PublishedValue struct {
+	Recipient AddressValue
+	Value     *CapabilityValue
```

Comment:
This implementation should work fore now, given that the published value is jut a Capability. For arbitrary values, which this might get refactored to in the future, the storage-related parts of the value are not quite complete e.g. transfer, deep removal, etc. are not supported.

Maybe add a TODO/NOTE warning above the`Value` field, that once the field is changed to type `Value`, `PublishedValue` needs to be extended and the remaining storage-related code must be implement.